src/dlgNotepad.cpp

```diff
@@ -49,7 +49,7 @@ dlgNotepad::~dlgNotepad()
 
 void dlgNotepad::save()
 {
-    QString directoryFile = mudlet::getMudletPath(mudlet::profileHomePath, mpHost->getName());
+    QString directoryFile = mudlet::getMudletPath(mudlet::profileHomePath, mpHost->getName(), QString());
```

Comment:
:hammer_and_wrench: Why the change to include an empty `QString()` as the third argument?  Both it and the second arguments have default values of `QString()` anyway...!

Reply:
:man_shrugging: inherited from the previous code. Removed.